app/Models/Thread.php

```diff
@@ -206,8 +206,8 @@ public static function resolutionTime()
         try {
             return static::join('replies', 'threads.solution_reply_id', '=', 'replies.id')
                 ->select(DB::raw('avg(datediff(replies.created_at, threads.created_at)) as duration'))
-                ->pluck('duration')
-                ->first();
+                ->first()
+                ->duration;
```

Comment:
Enlightn has a Larastan [NoUnnecessaryCollectionCall](https://github.com/nunomaduro/larastan/blob/master/docs/rules.md#nounnecessarycollectioncall) rule. This just defers the call to `first` to the query instead of the collection. I tested with mySQL and it's working fine.